### PR TITLE
Update geckodriver + Firefox versions

### PR DESCRIPTION
**Pull request by stephendonner**
Just noticed this, and figured we might want to be on the latest versions of both Firefox and Geckodriver.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ python:
   - 2.7
   - 3.6
 addons:
-  firefox: 53.0
+  firefox: 54.0
```
Comment (jrbenny35) [inline .travis.yml]: You can just do ```latest```.

see: https://docs.travis-ci.com/user/firefox

Comment (stephendonner) [inline .travis.yml]: Thanks, indeed.  I think (and @davehunt can confirm, though), that we probably want to pin.  I'm honestly not sure, so kept it the safe way, for now :-)

Comment (jrbenny35) [inline .travis.yml]: Using ```latest``` will keep up with only the releases, but I see your point.

Comment (davehunt) [inline .travis.yml]: This used to be `latest` but we had to pin while we were migrating to GeckoDriver/Marionette. Let's switch back to `latest` so we don't have to keep bumping it.

**Pull request merged**